app/controllers/images_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ImagesController < ApplicationController
+
+  before_action :authenticate_user!, :except => [:index, :show]
+  # permit(:title, :description, :picture)
+
+  def index
+    @images = Image.all
+  end
+
+  def new
+    @image = Image.new
+  end
+
+  def create
+    @image = current_user.images.build(image_params)
+    if @image.save
+      redirect_to images_path
+    else
+      render 'new'
+    end
+  end
+
+  def show
+    @image = Image.find(params[:id])
+  end
+
+  def destroy
+    @image = Image.find(params[:id])
+    if current_user.id === @image.user_id
+      @image.destroy
+      flash[:notice] = 'Image deleted successfully'
+      redirect_to images_path
```

Comment:
If you've got an if...else...end where the outcomes (a redirect) of both the if and else are the same, move the redirect to after end.